consensus nodes: add a section on statesyncing (#550)
celestiaorg/docs: docs/nodes/validator-node.md

\"$PRUNING_INTERVAL\"/" $HOME/.celestia-app/config/app.toml
```

### Syncing

By default, a consensus node will sync using block sync; that is request, validate
and execute every block up to the head of the blockchain. This is the most secure
mechanism yet the slowest (taking up to days depending on the height of the blockchain).

There are two alternatives for quicker syncing.

#### Statesync

> Statesync uses light client verification to verify state snapshots from peers

Reviewer: ```suggestion
State sync uses light client verification to verify state snapshots from peers
```

and then apply them. State sync relies on weak subjectivity; a trusted header
(specifically the hash and height) must be provided. This can be found by querying
a trusted RPC endpoint (/block). RPC endpoints are also required for retrieving
light blocks. These can be found in the docs here under the respective networks or
from the [chain-registry](https://github.com/cosmos/chain-registry).

In `$HOME/.celestia-app/config/config.toml`, set

```toml
rpc_servers = ""
trust_height = 0
trust_hash = ""
```

to their respective fields. At least two different rpc endpoints should be provided.
The more, the greater the chance of detecting any fraudulent behavior.

Once setup, you should be ready to start the node as normal. In the logs, you should
see: `Discovering snapshots`. This may take a few minutes before snapshots are found
depending on the network topology.

> #### Quicksync

Reviewer: ```suggestion
#### Quick sync
```

> Quicksync effectively downloads the entire `data` directory from a third-party provider

Reviewer: ```suggestion
Quick sync effectively downloads the entire `data` directory from a third-party provider
```
